internal/model/stocksfeed.go

```diff
@@ -0,0 +1,22 @@
+package model
+
+import (
+	"github.com/golang/protobuf/ptypes/timestamp"
+	"math/big"
+)
+
+type StocksFeed struct {
+	Id        big.Int             `gorm:"primary_key;column:id" json:"id"`
+	StockName string              `gorm:"column:stock_name" json:"stock_name" `
+	LTP       int                 `gorm:"column:ltp" json:"ltp" `
+	Open      int                 `gorm:"column:open" json:"open" `
+	High      int                 `gorm:"column:high" json:"high" `
+	Low       int                 `gorm:"column:low" json:"low" `
+	TradedAt  timestamp.Timestamp `gorm:"column:traded_at" json:"traded_at" `
```

Comment:
follow the same convention for timestamp.

Reply:
I have used timestamp.Timestamp in all my files